Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM phusion/baseimage:latest
+MAINTAINER Boik boik@tdohacker.org
+
+ENV HOME /root
+ARG DOMAIN
+ARG MODE=dev
+CMD ["/sbin/my_init"]
+
+WORKDIR $HOME
+RUN apt-get update \
+  && apt-get install -y rsync
+ADD . $HOME
+RUN chown root:root $HOME
+
+RUN ./extra/provision.sh -m $MODE -s `pwd`
+RUN [ "$MODE" = "prod" ] && rm /etc/nginx/sites-enabled/fbctf.conf || true
+
+COPY extra/http_fbctf.conf /etc/nginx/sites-available/http_fbctf.conf
```

Comment:
We already provide an nginx configuration, is this specifically for letsencrypt? We have built in letsencrypt support now, you can just use that.